test-suite/vos/run.sh

```diff
@@ -1,13 +1,24 @@
 #!/bin/bash
 set -e
 set -o pipefail
+# export COQBIN=../../bin
 export PATH="$COQBIN:$PATH"
 
-# Clean
-rm -f *.vo *.vos *.vok *.glob *.aux Makefile
+# requires "make coqlight coqbinaries tools"
+# can be interrupted after compilation of the prelude
 
-# Test building all vos, then all vok
-coq_makefile -R . TEST -o Makefile *.v
+checker() { # give as argument the name of the output and expected output
+  diff -q -a -u --strip-trailing-cr $1 $2 >&1
+  if [ $? -ne 0 ]; then
+    exit 1
+  fi
```

Comment:
```suggestion
  diff -q -a -u --strip-trailing-cr $1 $2 >&1 || exit 1
```
iiuc

Reply:
I wasn't sure what was the policy in case of errors, w.r.t printing the contents of the diff or not.

Reply:
My suggestion isn't supposed to change the behaviour so not sure what you mean